epub33/a11y-tech/index.html

```diff
@@ -1680,6 +1690,336 @@ <h4>Identify the pagination source</h4>
+						within each EPUB Content Document.</p>
+
+					<p>If there are cases where the logical reading order (how a reader would naturally read the
+						content) diverges from the default reading order, EPUB Creators can order the playback sequence
+						of <a data-cite="epub-33#sec-smil-seq-elem"><code>seq</code></a> and <a
+							data-cite="epub-33#sec-smil-par-elem"><code>par</code></a> elements in a <a
+							data-cite="epub-33#sec-overlay-docs">Media Overlays Document</a> [[EPUB-33]] to match the
+						logical order.</p>
+
+					<p>EPUB Creators need to use caution when making alterations, however, as other accessibility issues
+						can arise when the logical order does not match the default order. For example, the content may
+						not be accessible to users of assistive technologies when the order in the markup does not match
+						how the assistive technology reads the content. In these cases, using playback to create a
+						logical order can make the EPUB Publication fail WCAG conformance requirements.</p>
+
+					<p>One case where the logical may diverge from the reading order and remain accessible is in tables,
+						as assistive technologies typically allow users to choose whether to read by row or by
+						column.</p>
+				</section>
+
+				<section id="sync-003">
+					<h4>Identifying skippable structures</h4>
+
+					<p>Some content elements are not critical to read when following the primary narrative of a work,
+						and that would interrupt a user's concentration if they had to stop and listen to. Footnotes and
+						endnotes are examples of such content, as users may only want to come back and read this content
+						after finishing the EPUB Publication. The announcement of page break numbers can be similarly
+						annoying to readers.</p>
+
+					<p>EPUB 3's <a data-cite="epub-33#sec-media-overlays">Media Overlays feature</a> [[EPUB-33]] does
+						not allow Reading Systems to determine if playback sequences are skippable unless EPUB Creators
+						add additional semantics to the markup, however. EPUB Creators must use the <a
+							data-cite="epub-33#sec-epub-type-attribute"><code>epub:type</code> attribute</a> [[EPUB-33]]
+						to add semantics to <a data-cite="epub-33#sec-smil-seq-elem"><code>seq</code></a> and <a
+							data-cite="epub-33#sec-smil-par-elem"><code>par</code></a> elements [[EPUB-33]], thereby
+						allowing Reading Systems to provide users the option to skip their playback sequences.</p>
+
+					<p>EPUB Creators do not need to identify every structure to meet this requirement. The recommended
+						structures to identify for skippability are:</p>
+
+					<ul>
+						<li>Endnotes &#8212; use the <a data-cite="epub-ssv-11#endnotes"><code>endnotes</code>
+								semantic</a> for a group of notes or the <a data-cite="epub-ssv-11#endnote"
+									><code>endnote</code> semantic</a> [[EPUB-SSV-11]] for individual endnotes.</li>
```

Comment:
At first reading I did not realize that both the `endnotes` and the `endnote` values are defined in the same document (ie, SSV). Ie, that the reference to SSV is also valid for `endnotes`. Repeating the reference may be an overkill; maybe making it more explicit saying something like "both defined in [[...]]" might help.

This a repeated pattern here and in the part on skippability.

Reply:
I've rephrase to: "use the endnotes and endnote semantics [EPUB-SSV-11] for groups of notes and individual notes, respectively."

Same for footnotes.